text/1589-rustc-bug-fix-procedure.md

```diff
@@ -98,29 +98,31 @@ What follows is a template for tracking issues.
 
 ---------------------------------------------------------------------------
 
-This is the **summary issue** for the `YOUR_LINT_NAME_HERE`
-future-compatibility warning and other related errors. The goal of
-this page is describe why this change was made and how you can fix
-code that is affected by it. It also provides a place to ask questions
-or register a complaint if you feel the change should not be made. For
-more information on the policy around future-compatibility warnings,
-see our [breaking change policy guidelines][guidelines].
```

Comment:
This paragraph in the old template seems like a good intro to C-future-compatibility warnings in general (and it tells user that they can register a complaint). I'd keep it above "what is this lint about".

Reply:
I agree, I think losing this paragraph is a net loss. Remember that people are going to come to this bug straight from a rustc compiler error message and not really understand what is happening.

Reply:
i have no problem putting it back; my goal was to make the template reflect current practice, but I don't mind pushing back to keep things that would be improvements on current practice.